src/app/helpers/jwt.interceptor.spec.ts

```diff
@@ -1,45 +1,69 @@
-import {TestBed} from '@angular/core/testing';
-import {HttpClientTestingModule} from '@angular/common/http/testing';
-import {httpHandlerSpy, httpRequestSpy} from './spies';
-import {HttpHandler, HttpRequest} from '@angular/common/http';
-import {of} from 'rxjs';
-import {JwtInterceptor} from './jwt.interceptor';
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { httpHandlerSpy, httpRequestSpy } from 'src/app/helpers/spies';
+import { HttpHandler, HttpRequest } from '@angular/common/http';
+import { of } from 'rxjs';
+import { JwtInterceptor } from 'src/app/helpers/jwt.interceptor';
+import { AuthService } from 'src/app/services/auth.service';
 
 
 describe('JWTInterceptor', () => {
   let jwtInterceptor: JwtInterceptor;
+  let authService: AuthService;
+  let service;
+  let authServiceData;
+
+  authServiceData = {
+    currentUser: {
+      token: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MTAsImVtYWlsIjoib2dyZUB2aW1haWwyNC5jb20iLCJleHAiOjE1NjY2MzQxMDB9.5u2ylra-VwFb5qDjVsTVfcXPlGZlr9bGg54YyCwvkDA'
```

Comment:
@Manorlds-Eaglespark , I am not understanding why is this hardcoded token here. Can you please help to understand?